1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,79 @@
-// Write your code here
+const images = [
+  {
+    src: "images/photo-1637710133669-3dd2c9513041.jpeg",
+    alt: "nature picture",
+  },
+  {
+    src: "images/photo-1637710527019-b43708c651df.jpeg",
+    alt: "bird picture",
+  },
+
+  {
+    src: "images/photo-1644990978896-c86bd2e5fd2e.jpeg",
+    alt: "nature picture",
+  },
+
+  {
+    src: "images/photo-1625973385769-b9113cf6c2cf.jpeg",
+    alt: "see picture",
+  },
+];
+let autoForwardInt;
+let autoBackInt;
+const imagContainer = document.getElementById("imgContainer");
+const btnContainer = document.getElementById("btnContainer");
+const image = document.createElement("img");
+const forwardBtn = document.createElement("button");
+const backBtn = document.createElement("button");
+const autoForwardBtn = document.createElement("button");
+const autoBackBtn = document.createElement("button");
+const stopBtn = document.createElement("button");
+forwardBtn.innerHTML = "forward";
+backBtn.innerHTML = "back";
+stopBtn.innerHTML = "stop";
+autoBackBtn.innerHTML = "Auto Back";
+autoForwardBtn.innerHTML = "Auto Forward";
+imagContainer.appendChild(image);
+btnContainer.append(autoBackBtn, backBtn, stopBtn, forwardBtn, autoForwardBtn);
+let counter = 0;
+image.src = images[counter].src;
+image.alt = images[counter].alt;
+
```

Comment:
I'm not sure about you, but I hate it when buttons and things move around on the screen instead of staying in the same place. I added the below just to ensure all the images are the same size and the button bar doesn't jump around the screen. 
```
imagContainer.style.height = "500px";
imagContainer.style.width = "800px";
image.style.height = "500px";
image.style.width = "800px";
```